incubator/hnc/docs/user-guide/best-practices.md

```diff
@@ -75,20 +75,30 @@ kubectl hns describe foo
 
 Since HNC is controlled by regular Kubernetes objects, you can check your YAML
 files into source control and apply them to your cluster(s) via `kubectl apply
--f`. However, HNC does impose two restrictions on the order in which changes can
-be applied:
+-f`. You generally will only want to do this for your cluster-wide configuration
+(the `HNCConfiguration` object) and your full namespace configurations (the
+`HierarchyConfiguration` objects in each namespace), and not your subnamespaces
+(the `SubnamespaceAnchor` objects) since subnamespaces are mainly for
+unprivileged users, not Gitops flows.
+
+When applying your `HierarchyConfiguration` objects, HNC imposes two
+restrictions on the order in which changes can be applied:
 
 1. A namespace must exist before it can be referenced as the parent of another
-   namespace. However, the would-be parent’s `HierarchicalConfiguration` does
-   _not_ need to exist.
```

Comment:
Why is this removed?

Reply:
It just seemed redundant to me - I didn't think it would be a question that people would ask.

Reply:
wdyt?

Reply:
sg